identity/identity_test.go

```diff
@@ -111,7 +111,7 @@ func (s *CredentialsTestSuite) TestCompleteCredentialsFromEnvDefaultDomain(c *gc
 		"OS_PASSWORD":            "test-pass",
 		"OS_ACCESS_KEY":          "test-access-key",
 		"OS_SECRET_KEY":          "test-secret-key",
-		"OS_PROJECT_NAME":        "tenant-name",
+		"OS_PROJECT_ID":          "tenant-name",
```

Comment:
this isn't right - we need to test that either OS_PROJECT_NAME or OS_PROJECT_ID works.